src/dsp_tools/commands/xmlupload/transform_resource_and_values_for_upload.py

```diff
@@ -242,167 +225,257 @@ def _make_bitstream_file_value(bitstream_info: BitstreamInfo) -> dict[str, Any]:
             raise BaseError(f"Unknown file ending '{file_ending}' for file '{local_file}'")
 
 
-def _make_boolean_prop(
-    prop: XMLProperty, res_bn: BNode, prop_name: URIRef, permissions_lookup: dict[str, Permissions]
+def _make_simple_prop_graph(
```

Comment:
This and the function `_make_simple_value_graph` serialise a property when it follows the most basic structure.